packages/@okta/vuepress-site/docs/reference/api/apps/index.md

```diff
@@ -7622,3 +8597,151 @@ Okta updates a user's attributes in the application when the application is assi
 | Property         | Description                                                  | DataType                                                                    | Nullable | Unique | Readonly | Default |
 | ---------------- | ------------------------------------------------------------ | --------------------------------------------------------------------------- | -------- | ------ | -------- | --------- |
 | status   | Status of the setting     | `ENABLED`, `DISABLED` | FALSE    | FALSE   | FALSE    | `DISABLED`  |
+
+### Client Secret Request object
+
+#### Example
+
+```json
+{
+  "client_secret": "D0HxBn1FtTXeYC4cSBwWL_sPMztMT2t6Ei9n1QjO"
+}
+```
+
+#### Client Secret Request object properties
+
+| Property         | Description                                                  | DataType                                                                    | Nullable | Unique | Readonly | MinLength | MaxLength |
+| ---------------- | ------------------------------------------------------------ | --------------------------------------------------------------------------- | -------- | ------ | -------- | --------- | --------- |
+| client_secret    | Client secret                                                | String                                                                      | TRUE     | TRUE   | FALSE    |    14/32  |   100     |
```

Comment:
I wonder if it's better to use the `14` as the minimum and just have the little section with a special rule for specific token_endpoint_auth_method. Do we have other examples where we specify more than 1 value in a table like this anywhere else in the docs?

Reply:
No, I couldn't find one.

Reply:
updated